test/js/samples/bind-online/expected.js

```diff
@@ -11,14 +11,11 @@ import {
 
 function create_fragment(ctx) {
 	let dispose;
-	add_render_callback(ctx.onlinestatuschanged);
+	add_render_callback(ctx[1]);
 
 	return {
 		c() {
-			dispose = [
-				listen(window, "online", ctx.onlinestatuschanged),
-				listen(window, "offline", ctx.onlinestatuschanged)
-			];
+			dispose = [listen(window, "online", ctx[1]), listen(window, "offline", ctx[1])];
```

Comment:
It doesn't *really* matter, but the linebreaks were nice here.

Reply:
code-red breaks the line when it gets 'too long' which is a bit of a hand-wavey calculation (because it doesn't know, when calculating it, how much indentation will eventually be applied) — it would be easy enough to reduce the magic number involved in that calculation though

Reply:
Not convinced it's worth it, just calling out that the new version is a little awkward,

Also not gonna claim that I spend a lot of time looking at our generated JS and nitpicking the style of it.